lua/neogit/popups/tag/actions.lua

```diff
@@ -0,0 +1,161 @@
+local M = {}
+local git = require("neogit.lib.git")
+local client = require("neogit.client")
+local utils = require("neogit.lib.util")
+local FuzzyFinderBuffer = require("neogit.buffers.fuzzy_finder")
+
+--- Gets revisions
+---@return table
+local function get_revisions()
+  local revisions = git.cli["for-each-ref"].format('"%(refname:short)"').call():trim().stdout
+  for i, str in ipairs(revisions) do
+    revisions[i] = string.sub(str, 2, -2)
+  end
+  return revisions
+end
+
+--- Outputs a list of tags locally
+---@return table|nil List of tags.
+local function list_tags()
+  local revisions = git.cli.tag.args("-l").call():trim().stdout
+  if #revisions == 0 then
+    return nil
+  end
+  return revisions
+end
+
+function M.create_tag(popup)
+  local value = vim.fn.input { prompt = "Tag name: " }
```

Comment:
This can use `neogit.lib.input`: https://github.com/NeogitOrg/neogit/blob/08d23c7ae1e7c3c66802990d5aa16269bdd996a7/lua/neogit/lib/input.lua#L67-L69